modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/GeneOrderConservation.pm

```diff
@@ -460,4 +466,21 @@ sub _split_polyploid_goc {
     }
 }
 
+sub _write_n_tag {
+    my ($self, $mlss, $label, $scores) = @_;
+
+    my %distrib_hash;
+    foreach my $score ( values %$scores ) {
+        my $floor_score = int($score/25)*25;
```

Comment:
Same as in `AssignQualityScore.pm`. Furthermore, you have declared the same method twice in two runnables, so you should consider refactoring it in the `Utils` module, for instance (easier to maintain).